src/MCControl.cpp

```diff
@@ -29,6 +29,8 @@
 #include <RBDyn/FK.h>
 #include <RBDyn/FV.h>
 
+using highres_clock = std::chrono::high_resolution_clock;
```

Comment:
It's probably ok in this case (especially in the environments where this measurement matters) but it might be slight better to make sure we use a steady clock for the measurement (see https://github.com/jrl-umi3218/mc_rtc/blob/master/src/mc_control/mc_global_controller.cpp#L671)

This [page](https://en.cppreference.com/w/cpp/chrono/high_resolution_clock) suggests that the high_resolution_clock is not necessarily steady when using gcc's libstdc++ and for measuring this time I think it is important that the clock is steady.

Reply:
https://github.com/jrl-umi3218/mc_openrtm/pull/16/commits/f6319338d958a19df93e0a522f4c59661174daff changes the clock type to the steady clock.